### ISSUE TITLE
Landscape support

### ISSUE THREAD

**Issue by johnnyclem** [nickoneill/PermissionScope#92]
I'm trying to implement permissionscope in a landscape only app, and the dialog box spills off the page, making it impossible to use.  Can landscape support be added in the future? I'd write the PR myself, but UI is my weak point

**Comment by nickoneill** [comment]
Yeah, this seems doable for the future. I'll have to see how many we can fit vertically, or if we should go horizontal with the permission items.

Thanks! Happy to see a PR with a wip if you're feeling adventurous.

**Comment by bre7** [comment]
Might be a good idea to check out [OAStackView](https://github.com/oarrabi/OAStackView)/[TZStackView](https://github.com/tomvanzummeren/TZStackView) (UIStackView after PermissionScope drops support for iOS 8)

**Comment by harryworld** [comment]
+1 My app is landscape only

**Comment by bre7** [comment]
Force Portrait mode for PS' dialog ? http://stackoverflow.com/a/28220616/2124535
